internal/k8s/ako_init.go

```diff
@@ -43,24 +44,31 @@ import (
 	networkingv1 "k8s.io/api/networking/v1"
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
+	servicesapi "sigs.k8s.io/service-apis/apis/v1alpha1"
 )
 
 func PopulateCache() error {
+	var parentKeys []avicache.NamespaceName
 	var err error
-	avi_rest_client_pool := avicache.SharedAVIClients()
-	avi_obj_cache := avicache.SharedAviObjCache()
+	aviRestClientPool := avicache.SharedAVIClients()
+	aviObjCache := avicache.SharedAviObjCache()
 	// Randomly pickup a client.
-	if avi_rest_client_pool != nil && len(avi_rest_client_pool.AviClient) > 0 {
-		_, _, err = avi_obj_cache.AviObjCachePopulate(avi_rest_client_pool.AviClient, utils.CtrlVersion, utils.CloudName)
+	if aviRestClientPool != nil && len(aviRestClientPool.AviClient) > 0 {
+		_, parentKeys, err = aviObjCache.AviObjCachePopulate(aviRestClientPool.AviClient, lib.AKOControlConfig().ControllerVersion(), utils.CloudName)
```

Comment:
@abhishekbsingh: With latest HA code, we have changes over here and not camelcase changes. So we need to address this.